Formula/ola.rb

```diff
@@ -3,45 +3,35 @@ class Ola < Formula
   homepage "https://www.openlighting.org/ola/"
   url "https://github.com/OpenLightingProject/ola/releases/download/0.10.7/ola-0.10.7.tar.gz"
   sha256 "8a65242d95e0622a3553df498e0db323a13e99eeb1accc63a8a2ca8913ab31a0"
-  revision 2
+  revision 3
+  head "https://github.com/OpenLightingProject/ola.git"
 
   bottle do
     sha256 "cfd140e18776665bea8ff51d04f33db99d3925433d6c51a3dde8684bb9b408a9" => :catalina
     sha256 "9b6ba39db2eac356f2811ad76f3d6dae0e7b9d0f22a0dbc919fa0e4fda9cd639" => :mojave
     sha256 "949692cfbcf0408d27e523d858b38a778e812994f28f7790f09b1ceb6ec9187a" => :high_sierra
   end
 
-  head do
-    url "https://github.com/OpenLightingProject/ola.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "liblo"
   depends_on "libmicrohttpd"
   depends_on "libusb"
-  depends_on "numpy@1.16"
-  depends_on "protobuf@3.1"
-  depends_on "python@2" # protobuf@3.1 does not support Python 3
```

Comment:
@jonchang we still have some bugs with Python 3:
Issue: https://github.com/OpenLightingProject/ola/issues/1506
WIP PR: https://github.com/OpenLightingProject/ola/pull/1538

Reply:
Hopefully you will have a new release with better support soon, then :)